node/vanilla-quick-start/org.electronjs.ElectronQuickStart.yaml

```
@@ -62,6 +60,9 @@ modules:
       - cp -r electron-quick-start /app
       # Install the wrapper script to start it.
       - install -Dm 755 start-electron-quick-start.sh /app/bin/start-electron-quick-start
+
+      # To allow separate locales
+      - setup-separate-locales ${FLATPAK_DEST}/electron-quick-start/locales
```

https://github.com/flathub/org.electronjs.Electron2.BaseApp/pull/40#discussion_r1069560427
```suggestion
      - setup-separate-locales ${FLATPAK_DEST}/electron-quick-start/locales
```